src/sentry/conf/server.py

```diff
@@ -701,6 +701,7 @@ def create_partitioned_queues(name):
     'user': 'sentry.interfaces.user.User',
     'applecrashreport': 'sentry.interfaces.applecrash.AppleCrashReport',
     'breadcrumbs': 'sentry.interfaces.breadcrumbs.Breadcrumbs',
+    'contexts': 'sentry.interfaces.contexts.Contexts',
```

Comment:
I'm not sure if we need to call it the plural form (context to me implies any amount of context), but I dont have strong feelings either way.